static/vrm-interaction.js

```
@@ -0,0 +1,883 @@
+/**
+ * VRM 交互模块
+ * 负责拖拽、缩放、鼠标跟踪等交互功能
+ */
+
+// 确保 THREE 可用（支持 window.THREE 或全局 THREE）
+const THREE = window.THREE || (typeof THREE !== 'undefined' ? THREE : null);
+if (!THREE) {
+    console.error('[VRM Interaction] THREE.js 未加载，交互功能将不可用');
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)